introduction.md

```diff
@@ -6,3 +6,6 @@ Introducing myself to the Dojo, where my favourite Code School path is Ruby
 * I will never bring live blades on the tatami
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?